src/lib.rs

```diff
@@ -60,6 +63,39 @@ pub fn inspect(config: &Config) -> Result<(), InspectError> {
 
     if config.plain {
         println!("{}", output);
+    }
+    else if config.unpretty.starts_with("flowgraph"){
+        if let Some(path) = &config.input {
+            let input_path = PathBuf::from(path);
+            let mut output_path = PathBuf::new();
+            output_path.push(input_path.file_name().unwrap());
+            output_path.set_extension("png");
+
+            // Create a temporary file to dump out the plain output
+            let tmp_file_path = tmpfile()?;
+            {
+                let mut file = File::create(&tmp_file_path)?;
+                {
+                    file.write_all(output.as_bytes())?;
+                }
+            }
+
+            // For now setup the correct `dot` arguments to write to a png
+            let output_str = output_path.to_str()
+                .ok_or(InspectError::Generic(String::from("Failed to convert output path to string.")))?;
+            let input_str = tmp_file_path.to_str()
+                .ok_or(InspectError::Generic(String::from("Failed to convert temporary path to string.")))?;
+
+            let args = [
+                "-Tpng",
+                "-o", output_str,
+                input_str,
+            ];
+            log::info!("Writing \"{}\"...", output_str);
+
+            Command::new("dot").args(&args).spawn()?;
```

Comment:
There's a chance that we can show a pretty error message here I think.
Something like
```
Command::new("dot").args(&args).spawn().map_err(InspectError::MissingTool)?;
```
https://github.com/bheisler/criterion.rs/blob/bb8cedcc75033a006845f797d1d56e575d549e3c/plot/src/lib.rs#L1015

Reply:
Sounds good, I will have a look into adding errors and reporting them 👍